slicingTool/runProjectSlicingTool.sh

```diff
@@ -26,14 +34,18 @@ if [[ -n $DB && -n $USER && -n $PASS ]]
 then
     if [[ $USER =~ "curator" ]]
     then
-	echo Dropping $DB ...
-	mysql -u$USER -p$PASS -e "drop database if exists $DB"
-#	mysql -u$USER -p$PASS -e "drop database if exists ${DB}_myisam"
-#	mysql -u$USER -p$PASS -e "create database ${DB}_myisam"
-#	mysql -u$USER -p$PASS -e "drop database if exists gk_central"
-#	mysql -u$USER -p$PASS -e "create database gk_central"
+        # Attempts to find and use database $DB and if successful it is backed up
+        # before being dropped
+        if [[ mysql -u $USER -p$PASS -e 'use $DB' ]]
```

Comment:
Has this been tested? I think there is a space between `-u` and `$USER` and I'm not sure but I think that _may_ cause problems.

Reply:
A space is acceptable between -u and $USER but it can't be there for -p and $PASS.

Reply:
Ah, I didn't know there was a difference.